src/Categories.test.jsx

```diff
@@ -0,0 +1,39 @@
+import { render } from '@testing-library/react';
+
+import Categories from './Categories';
+
+import { categories } from '../__fixtures__/categories';
+
+jest.mock('react-redux');
+
+describe('Categories', () => {
+  context('with categories', () => {
+    it('render "category" buttons', () => {
+      const { getByRole } = render(
+        <Categories categories={categories} />,
+      );
+
+      categories.forEach((category) => {
+        expect(getByRole('button', { name: category.name })).toBeInTheDocument();
+      });
+    });
+
+    it('selected button, render category name with (V) ', () => {
+      const { getByText } = render(
+        <Categories categories={categories} selectedCategory={categories[0]} />,
+      );
+
+      expect(getByText(`${categories[0].name}(V)`)).toBeInTheDocument();
+    });
+  });
+
+  context('without categories', () => {
+    it('noting render', () => {
```

Comment:
```suggestion
    it('renders nothing', () => {
```

테스트 결과가 완전한 문장이 되도록 하면 좋을 것 같아요